src/vs/workbench/services/configurationResolver/test/electron-browser/configurationResolverService.test.ts

```diff
@@ -643,7 +643,7 @@ class MockCommandService implements ICommandService {
 	}
 }
 
-class MockQuickInputService implements IQuickInputService {
+export class MockQuickInputService implements IQuickInputService {
```

Comment:
SQL CARBON EDIT

Reply:
There is bad layering issue when calling this, so I will have to export this out to another file to be able to call it in notebookAction.test.ts